src/routes/cook.js

```diff
@@ -1,14 +1,119 @@
+/**
+ * @swagger
+ * tags:
+ *   name: Cooks
+ *   description: Cook management API for adding, updating, deleting and getting cooks.
+ */
 const express = require('express');
 
 const router = express.Router();
 
 const cookController = require('../controllers/cookController');
 const { validCook } = require('../validation/validator');
 
+/**
+ * @swagger
+ * /api/cook:
+ *   get:
+ *     summary: Get all cooks
+ *     tags: [Cooks]
+ *     responses:
+ *       200:
+ *         description: Success
+ */
 router.get('/', cookController.getAllCooks);
-router.post('/add',cookController.addCook);
+
+/**
+ * @swagger
+ * /api/cook/add:
+ *   post:
+ *     summary: Add a new cook
+ *     tags: [Cooks]
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Cook'
+ *     responses:
+ *       201:
+ *         description: Cook successfully added
+ *       400:
+ *         description: Invalid request
+ *       500:
+ *         description: Internal server error
+ */
+router.post('/add', cookController.addCook);
+/**
+ * @swagger
+ * /api/cook/{id}:
+ *   get:
+ *     summary: Get a cook by ID
+ *     tags: [Cooks]
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         required: true
+ *         schema:
+ *           type: string
+ *     responses:
+ *       200:
+ *         description: Success
+ *       404:
+ *         description: Cook not found
+ *       500:
+ *         description: Internal server error
+ */
 router.get('/:id', cookController.getCookById);
+/**
+ * @swagger
+ * /api/cook/{id}:
+ *   put:
+ *     summary: Update a cook by ID
+ *     tags: [Cooks]
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         required: true
+ *         schema:
+ *           type: string
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Cook'
```

Comment:
here is also cook_update if you look at my code in configuration/swagger.js you will find out